inference-engine/src/transformations/src/transformations/convert_precision.cpp

```diff
@@ -262,6 +265,17 @@ bool fuse_type_to_topk(std::shared_ptr<ngraph::Node> & node, ngraph::element::Ty
     return false;
 }
 
+bool fuse_type_to_ctc_greedy_decoder_seq_len(std::shared_ptr<ngraph::Node> & node, ngraph::element::Type to, size_t idx) {
+    if (auto ctc_decoder = as_type_ptr<opset6::CTCGreedyDecoderSeqLen>(node)) {
+        if (to == element::i32 || to == element::i64) {
```

Comment:
`CTCGreedyDecoderSeqLen` has two outputs. So you need to check which output needs to be changed.